src/main/scala/io/iohk/ethereum/utils/Config.scala

```diff
@@ -242,12 +242,10 @@ object TxPoolConfig {
 trait MiningConfig {
   val ommersPoolSize: Int
   val blockCacheSize: Int
-  val coinbase: Address
   val activeTimeout: FiniteDuration
   val ommerPoolQueryTimeout: FiniteDuration
   val headerExtraData: ByteString
   val miningEnabled: Boolean
-  val ethashDir: String
   val mineRounds: Int
```

Comment:
Both this and `miningEnabled` seem to be no longer used, should we maybe delete them?

Reply:
I guess deleted them is the best option.